ImageRater/MainWindow.xaml.cs

```diff
@@ -14,16 +13,30 @@ public partial class MainWindow : Window
     public MainWindow()
     {
         using (var dialog = new FolderBrowserDialog())
```

Comment:
I think it would be good if the folder dialog would get it's own function inside the window code behind (this file) which returns the selected folder path, to make the code better readable.